index.html

```diff
@@ -1,159 +1,259 @@
-            <h3>Skills</h3>
-          </li>
-          <li>Codekit</li>
-          <li>Codepen</li>
-          <li>Github</li>
-          <li>GitLab</li>
-          <li>terminal</li>
-        </ul>
-      </li>
-    </ul>
-
-  </section>
-</body>
-
-</html>
\ No newline at end of file
+    </section>
+    <section class="about" id="about">
+      <h2 class="aboutheader">About</h2>
+      <p class="introabout">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea
+        eiusmod
+      </p>
+      <input type="button" class="btn3" value="Get my CV" />
+      <ul class="grid-container">
+        <li class="aboutcard">
+          <ul class="grid-item1">
+            <li class="about-title">
+              <h3>Languages</h3>
+            </li>
+            <li>JavaScript (es5/es6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </li>
+        <li class="aboutcard">
+          <ul class="grid-item2">
+            <li class="about-title">
+              <h3>Frameworks</h3>
+            </li>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+            <li>Selenium</li>
+          </ul>
+        </li>
+        <li class="aboutcard">
+          <ul class="grid-item3">
+            <li class="about-title">
+              <h3>Skills</h3>
+            </li>
+            <li>Codekit</li>
+            <li>Codepen</li>
+            <li>Github</li>
+            <li>GitLab</li>
+            <li>terminal</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
+    <form action="https://formspree.io/f/mayaljan" method="post" id="form">
+      <fieldset class="contact">
+        <h2 class="contactheader">Get Started</h2>
+        <p class="intro">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna.
+        </p>
+        <label for="name"></label>
+        <input
+          type="text"
+          class="name"
+          id="name"
+          name="name"
+          placeholder="   Your Name"
+          maxlength="30"
+          required
+        />
+        <br />
+        <label for="email"></label>
+        <input
+          type="email"
+          class="name"
+          id="email"
+          name="email"
+          placeholder="   Your Email"
+          required
+        />
+        <br />
+        <label for="message"></label>
+        <textarea
+          name="message"
+          class="message"
+          id="message"
+          placeholder="   Enter your message here..."
+          maxlength="500"
+          required
+        ></textarea>
+        <input type="submit" class="btn1" value="Start collaboration" />
```

Comment:
- [ ] [OPTIONAL] Hi, your page is styled really well according to the Figma template. One thing I've noticed is that your button has a light border around it. if you look keenly at the Figma design you'll notice there is no border around the submit button in the form section. I've set this as optional but please implement this 🙏.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/144142395-48cb4a8a-00bb-49d8-a5ed-dd974d6555bc.png)

## Figma Design 
![image](https://user-images.githubusercontent.com/18730106/144142219-786c2f2b-d2fb-4712-b3b4-8ed0adae04d9.png)